tasks/watch.js

```diff
@@ -13,9 +13,10 @@ module.exports = {
       '.flowconfig',
     ],
     tasks: [
-      'flowbin:status',
       'build-tests',
-      'mochaTest',
+      'mochaTest:unit',
+      'mochaTest:functional',
```

Comment:
I view `npm start` as a highly iterative development loop. It needs to be as fast as possible otherwise it's not so useful (and `npm test` is a better replacement). Thus, I think we should not run the functional tests in this loop. However, if you feel like they would still be useful then I would request to exclude them by default and include them only when a well-documented environment variable is present.